lib/northstar-client.js

```diff
@@ -10,12 +12,55 @@ const VERSION = require('../package.json').version;
 class NorthstarClient {
 
   constructor(options) {
-    this.options = options;
+    this.options = options || {};
     this.VERSION = VERSION;
+
+    // TODO: actually parse options.
+    this.options.baseURI = 'https://northstar-qa.dosomething.org/v1/users';
```

Comment:
Should this be in a config file, out of the repo? Perhaps support for ENV flags for test vs production as I suspect production will not want to point at `northstar-qa`?